src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
simply adding `@final since version 3.4`
generally speaking, no need to make them really final in 4.0: that won't provide any specific value and would only prevent legit uses with e.g. lazy proxies/mocks